terraform/mysql.tf

```diff
@@ -0,0 +1,34 @@
+resource "azurerm_mysql_server" "main" {
+  name                = "${var.prefix}-mysql-server"
```

Comment:
Error: "name" did not match regex "^[0-9a-z][-0-9a-z]{1,61}[0-9a-z]$"

  on mysql.tf line 1, in resource "azurerm_mysql_server" "main":
   1: resource "azurerm_mysql_server" "main" {



Error: "name" can contain only lowercase letters, numbers, and '-', but can't start or end with '-', and must be at least 3 characters and no more than 63 characters long.

  on mysql.tf line 1, in resource "azurerm_mysql_server" "main":
   1: resource "azurerm_mysql_server" "main" {

Reply:
the variable was an name with uppercase